### ISSUE TITLE
There is already shadcn-svelte

### ISSUE THREAD

**Issue by mcmxcdev** [aidan-neel/neel-ui#1]
Maybe you are not aware of it, but there is already the quite popular https://github.com/huntabyte/shadcn-svelte

Some of the documentation pages like https://ui.aidan-neel.com/docs/components/dialog and https://shadcn-svelte.com/docs/components/dialog feel almost identical.

I would say the Svelte ecosystem would benefit from not having too many duplicated libraries that then run out of (much appreciated!) contributors time and suffer from maintenance issues.

Maybe you can reconsider creating essentially the same thing as shadcn-svelte and instead contribute to that project to become even more amazing!

**Comment by aidan-neel** [comment]
i am very aware of shadcn-svelte, the purpose of this project was a learning experience for me. im actually working on the v1 release which will differ a lot more from shadcn-svelte, between the docs, and the actual components stylings and transitions for example. 

the main difference is it not using primitives from bits/melt like shadcn-svelte does. every component is simple and you can go into the files you're importing directly and edit them way easier than you could with shadcn-svelte.

tldr: v1 will differentiate it from shadcn-svelte a lot

**Comment by mcmxcdev** [comment]
I still think the time would be more valuable spent on shadcn-svelte/bits-ui/melt-ui, but I wish you all the best for the success of your UI framework!
